pkg/cmd/cli/restore/describe.go

```diff
@@ -80,6 +80,11 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 					fmt.Fprintf(os.Stderr, "error getting PodVolumeRestores for restore %s: %v\n", restore.Name, err)
 				}
 
+				// Change phase to "Deleting" if deletionTimestamp is marked.
+				if !restoreList.Items[i].DeletionTimestamp.IsZero() {
+					restoreList.Items[i].Status.Phase = "Deleting"
```

Comment:
I don't think this will work, since "Deleting" isn't a valid value for this field. A better place to do this would be here:
https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/util/output/restore_describer.go#L46

Also, I don't think we want to lose the actual value of the field (i.e. completed vs. failed), so maybe in restore_describe.go, something like this:
```
	if !restore.DeletionTimestamp.IsZero() {
		phase = phase + " (Deleting)"
	}
```

Reply:
Make sense.